test/blackbox-tests/test-cases/doc-browser.t/dune-project

```diff
@@ -0,0 +1,4 @@
+(lang dune 3.7)
```

Comment:
Let's use 3.8 here.

Reply:
This is now 3.10

Reply:
It doesn't matter. If the test works with 3.7 and is not about a 3.7-versioned feature it's fine as is.